go.mod

```diff
@@ -4,6 +4,6 @@ go 1.13
 
 require (
 	github.com/gorilla/mux v1.7.3
-	github.com/mattermost/mattermost-server v0.0.0-20190927121038-340287890a78
+	github.com/mattermost/mattermost-server/v5 v5.18.0-rc.test
```

Comment:
`v5.18.0-rc.test` Huh?

Reply:
That's what it auto-resolves to, if I import `/v5` and `go mod tidy` with 1.13.x. Have you tried converting the workflow plugin to `/v5` yet?

Reply:
No I have not. You would think it would avoid RCs...

Reply:
Oh maybe because there isn't a release with `/v5` yet 😆